onnx/defs/reduction/defs.cc

```diff
@@ -134,8 +134,10 @@ std::function<void(OpSchema&)> ArgReduceDocGenerator(const char* name) {
   return [=](OpSchema& schema) {
     std::string doc = R"DOC(
 Computes the indices of the {name} elements of the input tensor's element along the 
-provided axis. The resulted tensor has the same rank as the input if keepdims equal 1.
+provided axis. The resulted tensor has the same rank as the input if keepdims equal 1. 
 If keepdims equal 0, then the resulted tensor have the reduced dimension pruned. 
+If select_last_index is True (default False), the index of the last occurence of the {name} 
+is selected if the {name} appears more than once in the input. 
```

Comment:
We should also add explicitly what is the behavior when `select_last_index=False` (the default) in this blurb as well.